apps/camera/locales/camera.en-US.properties

```diff
@@ -86,3 +86,11 @@ Video-Mode = Video Mode
 options = Options
 open-gallery = Open Gallery
 preview = Preview
+
+#The following strings are spoken by screen readers and not shown on the screen.
+flash-on-button.ariaLabel = Flash On
+flash-off-button.ariaLabel = Flash Off
+flash-auto-button.ariaLabel = Flash Auto
+toggle-camera-rear-button.ariaLabel = Rear camera
+toggle-camera-front-button.ariaLabel = Front camera
```

Comment:
@eeejay , originally I thought so too, but the labels above are consistent with the graphics. The buttons in any time display the current state and not the one it will be switching too.